sw/device/lib/testing/test_coverage.c

```diff
@@ -0,0 +1,84 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/uart.h"
+#include "sw/vendor/llvm_clang_rt_profile/compiler-rt/lib/profile/InstrProfiling.h"
+#include <stdint.h>
+
+/**
+ * When the linker finds a definition of this symbol, it knows to skip loading
+ * the object which contains the profiling runtime’s static initializer.
+ */
+int __llvm_profile_runtime;
+
+/**
+ * Start and end of `.init_array` section where the LLVM profiling runtime
+ * initialization function resides, defined in `flash_link.ld`.
+ */
+extern void (*_init_array_start[])(void);
```

Comment:
C decl syntax is awful. Maybe consider writing a typedef like
```c
typedef void (*init_t)(void);
```
or something...?

Reply:
Moved to `flash_crt.S`.